test/powershell/engine/Basic/Encoding.Tests.ps1

```diff
@@ -72,4 +74,28 @@ Describe "File encoding tests" -Tag CI {
             $bytes -join "-" | should -Be ($Expected -join "-")
         }
     }
+
+    Context "Parameter 'Encoding' should accept numeric and string Ids" {
+        BeforeAll {
+            $testValue = "ф"
+            if ($IsWindows) {
+                $expectedBytes = 244,13,10 -join "-"
```

Comment:
Add a comment about what the characters are

Reply:
Added.